src/main/java/pl/javastart/jjdind49_katibar_zad20_1/UserController.java

```diff
@@ -0,0 +1,57 @@
+package pl.javastart.jjdind49_katibar_zad20_1;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import java.util.List;
+
+@Controller
+public class UserController {
+
+    private final UsersRepo usersRepo;
+
+    public UserController(UsersRepo usersRepo) {
+        this.usersRepo = usersRepo;
+    }
+
+    @RequestMapping("/repo")
+    public String startPOST(@RequestParam String firstName, @RequestParam String lastName, @RequestParam Integer age) {
+        if (firstName != null) {
+            createNewUser(firstName, lastName, age);
+            return "redirect:/success.html";
+        } else {
+            return "redirect:/err.html";
+        }
+    }
```

Comment:
- nazwa `startPOST` nie jest zapisana jako `camelCase`
- nazwa `startPOST` niewiele mówi
- dlaczego adres to `/repo`?
- czy `firstName` faktycznie może być nullem? W jakim wypadku?